ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
+  if (brand && brand.length) {
+    return <img className={c('brand')} src={brand} />;
+  } else {
+    const firstLetter = name.split('')[0];
```

Comment:
We don't need to split the string to access its first character, we can directly do `name[0]`. We should uppercase it btw.